azimuth_caas_operator/tests/utils/test_ansible_runner.py

```diff
@@ -186,7 +186,7 @@ def test_get_job_env_configmap(self):
     '
   extravars: "cluster_deploy_ssh_public_key: fakekey\\ncluster_id: fakeclusterID1\\n\\
     cluster_image: testimage1\\ncluster_name: test1\\ncluster_ssh_private_key_file:\\
-    \\ /var/lib/caas/ssh/id_ed25519\\ncluster_type: type1\\nfoo: bar\\nnested:\\n  baz:\\
+    \\ /var/lib/caas/ssh/id_ed25519\\ncluster_type: type1\\nfoo: boo\\nnested:\\n  baz:\\
```

Comment:
This is cool, but its really hard to read the intent in the test (which is totally my fault when I wrote it).

Could we please add some extra checks with comments, something like this after the yaml assertion:

```
# check cluster_image comes from cluster template
self.assertEqual("testimage1", config["data"]["extravars"]["cluster_image"])
# check very_random_int comes from the cluster extravars
...
# check foo is correctly overridden by extraVarOverrides
...
```